client/components/snackbar/snackbar.js

```diff
@@ -0,0 +1,45 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import styles from './snackbar.sass'
+
+class Snackbar extends PureComponent {
```

Comment:
Just to confirm: A Snackbar is a notification that appears at the bottom of the screen, as seen in http://www.material-ui.com/#/components/snackbar ?

Reply:
Yep, that's exactly the inspiration.